.github/workflows/release-published.yml

```diff
@@ -1,80 +1,52 @@
 name: Release Extension to Sonatype
 
 on:
+  workflow_dispatch:
   release:
     types: [published]
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'Release Tag'
-        required: true
 
 jobs:
-  setup:
-    name: Setup
-    runs-on: ubuntu-latest
-    outputs:
-      tag: ${{ steps.collect-data.outputs.tag }}
-      version: ${{ steps.collect-data.outputs.version }}
-    steps:
-      - name: Collect Data
-        id: collect-data
-        uses: actions/github-script@v6
-        with:
-          script: |
-            let tag;
-            if (context.payload.inputs) {
-              tag = context.payload.inputs.tag;
-            } else {
-              tag = context.payload.release.tag_name;
-            }
-
-            let version = tag.replace(/^v/, "");
-
-            core.setOutput("tag", tag);
-            core.setOutput("version", version);
-
-      - run: |
-          echo "Publishing version ${{ steps.collect-data.outputs.version }} from ${{ steps.collect-data.outputs.tag }}"
+  release:
+    uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.5.2
+    secrets: inherit
 
-  deploy_maven:
-    name: Deploy to Maven
-    needs: [ setup ]
-    runs-on: ubuntu-latest
+  deploy_xsd:
```

Comment:
@jandroav I don't think we need to publish XSDs for this

Reply:
Copy paste 🫣